eth/backend.go

```diff
@@ -590,17 +590,21 @@ func (s *Ethereum) stopAnnounce() error {
 	return nil
 }
 
-func (s *Ethereum) StartProxyHandler() error {
+func (s *Ethereum) StartProxiedValidatorEngine() error {
 	if istanbul, ok := s.engine.(consensus.Istanbul); ok {
-		return istanbul.StartProxyHandler()
+		if istanbul.IsProxiedValidator() {
```

Comment:
should we return an error if we're trying to start when not a proxied validator?
Always better to  "fail fast"